packages/server/src/fhir/operations/conceptmaptranslate.ts

```diff
@@ -0,0 +1,182 @@
+import { parseInputParameters, sendOutputParameters } from './utils/parameters';
+import { Operator, allOk, badRequest, notFound } from '@medplum/core';
+import { Request, Response } from 'express';
+import { getOperationDefinition } from './definitions';
+import { CodeableConcept, Coding, ConceptMap, ConceptMapGroup, OperationOutcome } from '@medplum/fhirtypes';
+import { getAuthenticatedContext } from '../../context';
+import { sendOutcome } from '../outcomes';
+
+const operation = getOperationDefinition('ConceptMap', 'translate');
+
+type ConceptMapTranslateParameters = {
```

Comment:
Curious why `type` over `interface`?

Not a big deal, but hopefully as a team we can formulate a framework on how to choose.

In general, I've defaulted to `interface` in the past based on recommendations from the TypeScript's page on performance: https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections

Reply:
I was going off of [this line of thinking](https://blog.logrocket.com/types-vs-interfaces-typescript/#when-use-types-vs-interfaces), that `type` is more expressive and in theory works exactly the same for simple cases like this one.  I see the performance argument, but my read was that it only mattered when doing definition merging or extending, which isn't generally a major use case from what I've seen